lib/debug/repl.cpp

```diff
@@ -427,6 +453,46 @@ void Repl::addSpiCommands(const std::uint8_t bus)
   }
 }
 
+void Repl::addAccelerometerCommands(const std::uint8_t bus, const std::uint8_t device_address)
+{
+  const auto optional_i2c = io::HardwareI2c::create(logger_, bus);
+  if (!optional_i2c) {
+    logger_.log(core::LogLevel::kFatal, "Failed to create I2C instance on bus %d", bus);
+    return;
+  }
+  const auto i2c = std::make_shared<io::HardwareI2c>(*optional_i2c);
+  if (device_address != 0x19) {
```

Comment:
Where does this number come from? Should it be given a name and stored somewhere?

Reply:
pretty sure you have a` kDeviceAddress` already to check this @Platiniom64 (also since these are global, and we are adding more sensors, the name should be changed to `kDefaultAccelerometerAddress` to avoid confusion later)